### ISSUE TITLE
create_constant for arbitrary values

### ISSUE THREAD

**Issue by soumyac1999** [flexflow/FlexFlow#588]
Currently `create_constant` supports creating tensor with all values having the same value. It would be good to have one which can assign arbitrary constants. Something like:
```python
x0 = np.rand(10, 5)
c0 = create_constant(x0)

x1 = [[1,2], [3,4]]
c1 = create_constant(x1)

# below is probably already supported but adding for completeness
x2 = 5
c2 = create_constant(x2)
```

**Comment by jiazhihao** [comment]
@soumyac1999 Can you try and see if `attach_numpy_array` does what you need? It allows you to attach an arbitrary numpy array to a tensor. See examples at https://github.com/flexflow/FlexFlow/blob/master/examples/python/native/tensor_attach.py

**Comment by soumyac1999** [comment]
Not really. In Keras, `ffmodel` is not created until `model.compile` is called so I wouldn't be able to create tensors which could work as inputs to keras functions.
(This might related to #262 and #414.)

**Comment by jiazhihao** [comment]
A potential workaround is keeping this tensors in the Python layer (i.e., keeping pointers to the numpy arrays in the Python module) and attaching these numpy arrays to FlexFlow tensors after `model.compile`.

**Comment by eddy16112** [comment]
> A potential workaround is keeping this tensors in the Python layer (i.e., keeping pointers to the numpy arrays in the Python module) and attaching these numpy arrays to FlexFlow tensors after `model.compile`.

It is better not to use attach because the attach operation is not supported from the control replicated top level task. Instead the `set_tensor` should work after the `model.compile` is called.

**Comment by lockshaw** [comment]
I believe this is getting fixed (at least the feature is getting added, if not through this interface) in #771. @soumyac1999 Correct?
